<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>PG-ARM-COM</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.png" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/remixicon/remixicon.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: OnePage - v4.2.0
  * Template URL: https://bootstrapmade.com/onepage-multipurpose-bootstrap-template/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body>



<!-- ======= Header ======= -->
  <header id="header" class="fixed-top">
    <div class="container d-flex align-items-center justify-content-between">

      <h1 class="logo"><a href="index.html"></a></h1>
      <!-- Uncomment below if you prefer to use an image logo -->
      <!-- <a href="index.html" class="logo"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->

      <nav id="navbar" class="navbar">
        <ul>

          <li><a class="nav-link scrollto active" href="#hero">Home</a></li>
          <li><a class="nav-link scrollto" href="#motivation">Motivation</a></li>
          <li class="dropdown"><a href="#strategy"><span>Methods</span> <i class="bi bi-chevron-down"></i></a>
            <ul>
              <li><a href="#data">Data</a></li>
              <li><a href="#PPIprediction">Protein-Protein Interaction Prediction</a></li>
            </ul>
          </li>
		  <li><a class="nav-link scrollto" href="#results">Results</a></li>
          <li><a class="nav-link scrollto" href="#conclusions">Conclusions</a></li>
          <li><a class="nav-link scrollto" href="#minute-video">One-Minute Video</a></li>
          <li><a class="nav-link scrollto" href="#authors">Authors</a></li>
          <li><a class="nav-link scrollto" href="#funding">Funding</a></li

        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->



<!-- ======= Hero Section ======= -->
  <section id="hero" class="d-flex align-items-center">
    <div class="container position-relative" data-aos="fade-up" data-aos-delay="100">

      <div class="row justify-content-center">
        <div class="col-xl-12 col-lg-12 text-center">
          <h1>Pattern-Guided Association Rule Mining for Complex Ontology Alignment </h1>
          <h2>Beatriz Lima, Daniel Faria, Catia Pesquita</h2>
          <p>LASIGE, Faculdade de Ciências, Universidade de Lisboa</p>
        </div>
      </div>

      <div class="text-center">
        <a href="https://liseda-lab.github.io/PG-ARM-COM/paper324.pdf" class="btn-get-started scrollto"> Short Paper</a>
      </div>

     <div class="row icon-boxes">
	 
        <div class="col-md-6 col-lg-4 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="200">
          <div class="icon-box">
			<center> 
			  <h3 class="title">Biomedical knowledge graphs share, reuse and import data from each other routinely... </h4>
			  <img src="assets/img/Interoperability.png" class="img-fluid" alt="" width="350" height="350"> 
			 </center>
          </div>
        </div>

        <div class="col-md-6 col-lg-4 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="300">
          <div class="icon-box">
		    <center> 
			  <h3 class="title"> ... This can be a potential source of <strong>data leakage</strong> for machine learning applications using these knowledge graphs, creating predictive models with overestimated performance values. </h3>
			  <img src="assets/img/leakage.png" class="img-fluid" alt="" width="200" height="200"> 
			</center>
          </div>
        </div>

        <div class="col-md-6 col-lg-4 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="zoom-in" data-aos-delay="400">
          <div class="icon-box">
		    <center> 
			  <h3 class="title"> To investigate potential data leakage, we simulate a realistic training situation, where everything in the training set occurs before everything in the test set. </h3>
			  <img src="assets/img/TemporalData.png" class="img-fluid" alt="" width="300" height="300"> 
			</center>
          </div>
        </div>

      </div>

    </div>
  </section><!-- End Hero -->

<main id="main">



<!-- ======= About Section ======= -->
  <section id="motivation" class="about">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Motivation </h2>
        </div>

        <div class="row content">

          <div class="col-lg-6">
            <p style="text-align: justify;">
            As machine learning is increasingly being used, concerns about data leakage have been raised.
			</p>
			<p style="text-align: justify;">
			<a href="https://arxiv.org/abs/2003.08001" >Leakage</a>  occurs when information about the target of a data mining problem that should not be legitimately available to mine from is introduced,
			and it can lead to overestimation of the model’s performance. 
			In biomedical applications, such as protein-protein interaction (PPI) prediction, data leakage can also be an issue since multiple databases and resources reuse the same sources of information.  
            </p>
          </div>

          <div class="col-lg-6 pt-4 pt-lg-0">
            <p style="text-align: justify;">
            The majority of PPI prediction methods that are based on knowledge graphs (KGs) explore the Gene Ontology (GO) KG, composed of the <a href="http://geneontology.org/" >GO</a> and 
			<a href="http://geneontology.org/docs/go-annotations/">GO annotations</a>. 
			Since GO annotations can be based on the automated processing of other data sources, the same information that is used to support a PPI in a 
			<a href="https://string-db.org/cgi/input?sessionId=bHnvrRHPzilD&input_page_show_search=on">database</a> to 
			also be used to establish a GO annotation for the proteins.
			</p>
			<p style="text-align: justify;">
			We investigated potential data leakage between the GO KG and the STRING database in the task of PPI prediction, 
			by comparing performance on unseen interactions.
            </p>
          </div>

        </div>

      </div>
  </section><!-- End About Section -->



<!-- ======= Data Section ======= -->
  <section id="data" class="services section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Knowledge Graphs and Data Resources</h2>
          <div class="row justify-content-center">
            <div class="col-8" >
              <span> PPI prediction is cast as a classification task that takes as input the GO KG and a set of protein pairs.
					The first step of our approach is using historical data to build the PPI datasets.   
					Then we use the GO KG and the protein pairs to predict interactions using several machine learning algorithms. 
              </span>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="col-lg-6" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box iconbox-blue">
              
			  <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,521.0016835830174C376.1290562159157,517.8887921683347,466.0731472004068,529.7835943286574,510.70327084640275,468.03025145048787C554.3714126377745,407.6079735673963,508.03601936045806,328.9844924480964,491.2728898941984,256.3432110539036C474.5976632858925,184.082847569629,479.9380746630129,96.60480741107993,416.23090153303,58.64404602377083C348.86323505073057,18.502131276798302,261.93793281208167,40.57373210992963,193.5410806939664,78.93577620505333C130.42746243093433,114.334589627462,98.30271207620316,179.96522072025542,76.75703585869454,249.04625023123273C51.97151888228291,328.5150500222984,13.704378332031375,421.85034740162234,66.52175969318436,486.19268352777647C119.04800174914682,550.1803526380478,217.28368757567262,524.383925680826,300,521.0016835830174"></path>
                </svg>
                <i class="bi bi-diagram-3"></i>
              </div>
			  
              <h4><a href="">Gene Ontology Knowledge Graph</a></h4>
			  <p>
              <a href="http://geneontology.org/" >GO</a> describes protein function with respect to three semantic aspects: biological processes, cellular components, and molecular functions. 
              GO and the annotations that link proteins to GO classes make up a KG.
              We obtained archived versions of the GO and GO annotations in 2015, 2017 and 2019 from the <a href=http://release.geneontology.org/ >GO Data Archive</a>.
			  </p>
              
			  <p> <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#GOKG"> See GO KG <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i></a> </p>
                <div id="GOKG" class="collapse">
                  <img src="assets/img/GO.png" class="img-fluid" alt="" style="padding-top:20px" width="550" height="550">
                </div>
				
            </div>
          </div>

          <div class="col-lg-6" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box iconbox-yellow">
              
			  <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,582.0697525312426C382.5290701553225,586.8405444964366,449.9789794690241,525.3245884688669,502.5850820975895,461.55621195738473C556.606425686781,396.0723002908107,615.8543463187945,314.28637112970534,586.6730223649479,234.56875336149918C558.9533121215079,158.8439757836574,454.9685369536778,164.00468322053177,381.49747125262974,130.76875717737553C312.15926192815925,99.40240125094834,248.97055460311594,18.661163978235184,179.8680185752513,50.54337015887873C110.5421016452524,82.52863877960104,119.82277516462835,180.83849132639028,109.12597500060166,256.43424936330496C100.08760227029461,320.3096726198365,92.17705696193138,384.0621239912766,124.79988738764834,439.7174275375508C164.83382741302287,508.01625554203684,220.96474134820875,577.5009287672846,300,582.0697525312426"></path>
                </svg>
                <i class="bi bi-server"></i> 
              </div>
              
			  <h4><a href="">Protein-Protein Interaction Datasets</a></h4>
              <p> 
			  The PPI datasets were obtained from the <a href=https://string-db.org >STRING Database</a>, one of the largest available 
			  PPI databases. 
			  We built several PPI datasets using three archived versions of the STRING database (v9.1, v10, and v10.5) and the current version (v11). 
			  For the current version, we created three datasets each excluding protein pairs present in each of the older versions.
			  </p>
			  
              <p> <a data-bs-toggle="collapse" class="collapsed" data-bs-target="#PPIdatasets"> See the number of interactions in each version of the STRING database <i class="bx bx-chevron-down icon-show"></i><i class="bx bx-chevron-up icon-close"></i></a> </p>
                
				<div id="PPIdatasets" class="collapse">
				
				  <style>
		            #table_datasets{
		              padding-top: 25px; 
					  padding-bottom: 25px; 
		            }
                  </style>
				
                  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
                  <script type="text/javascript">
                  google.charts.load('current', {'packages':['table']});
                  google.charts.setOnLoadCallback(drawTable);
                  function drawTable() {
                  var data = new google.visualization.DataTable();
                  data.addColumn('string', 'STRING version');
                  data.addColumn('string', 'Date');
                  data.addColumn('number', 'Positive pairs');
                  data.addRows([
                  ['v9.1', '04/2015', 12681],
                  ['v10', '05/2017', 26863],
                  ['v10.5', '01/2019', 31384],
                  ['v11 (excluding pairs in v9.1)', '10/2020', 41227],
                  ['v11 (excluding pairs in v10)', '10/2020', 31642],
                  ['v11 (excluding pairs in v10.5)', '10/2020', 23571]
                  ]);
                  var table = new google.visualization.Table(document.getElementById('table_datasets'));
                  table.draw(data, {showRowNumber: false, width: '70%', height: '100%'});}
                  </script>
				
                  <div id="table_datasets"></div>
				
				  <p style="text-align: justify;"> We considered the following criteria to select protein pairs from STRING:</p> 
				  <p style="text-align: justify;"> (i) each protein must be annotated with the GO;</p> 
				  <p style="text-align: justify;"> (ii) protein interactions must be experimentally determined or from curated databases;</p> 
				  <p style="text-align: justify;"> (iii) interactions must have a confidence score above 950 to retain only high confidence interactions.</p> 
				  <p style="text-align: justify;"> We employed random sampling to create negative pairs composed of the human proteins present in the positive pairs but without any STRING interactions 
				  between them, building a balanced dataset. </p> 
				
				</div>
			  
            </div>
			
          </div>

        </div>

      </div>
  </section><!-- End Evaluation Section -->
  
  
  
<!-- ======= PPI Prediction Section ======= -->
  <section id="PPIprediction" class="services section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Protein-Protein Interaction Prediction</h2>
          <div class="row justify-content-center">
            <div class="col-8" >
			 <span> 
			 We follow the setup in <a href="https://openreview.net/pdf/c1c08929d05e8424a472d8b1e5d837a02e5ea6ae.pdf">evoKGsim+</a>, available on <a href="https://github.com/liseda-lab/evoKGsim" >GitHub</a>, that predicts relations 
			 between KG entity pairs that are not encoded in the graph using similarity-based semantic representations. 
			 </span>
            </div>
          </div>
        </div>

        <div class="row">

          <div class="col-lg-4 col-md-6" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box iconbox-blue">
			  <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,521.0016835830174C376.1290562159157,517.8887921683347,466.0731472004068,529.7835943286574,510.70327084640275,468.03025145048787C554.3714126377745,407.6079735673963,508.03601936045806,328.9844924480964,491.2728898941984,256.3432110539036C474.5976632858925,184.082847569629,479.9380746630129,96.60480741107993,416.23090153303,58.64404602377083C348.86323505073057,18.502131276798302,261.93793281208167,40.57373210992963,193.5410806939664,78.93577620505333C130.42746243093433,114.334589627462,98.30271207620316,179.96522072025542,76.75703585869454,249.04625023123273C51.97151888228291,328.5150500222984,13.704378332031375,421.85034740162234,66.52175969318436,486.19268352777647C119.04800174914682,550.1803526380478,217.28368757567262,524.383925680826,300,521.0016835830174"></path>
                </svg>
                <i class="bi bi-ui-checks"></i>
              </div>
			  <h4><a href="">Similarity Computation</a></h4>
              <p  style="text-align: justify;"> Employ three KG-based semantic similarity measures to compute semantic similarity:
			  (i) two taxonomic measures (<a href="http://www.cs.cmu.edu/afs/cs.cmu.edu/project/jair/pub/volume11/resnik99a.pdf" >ResnikMax</a>, <a href="https://webpages.ciencias.ulisboa.pt/~fjcouto/files/workshop%20cpesquita-ismb2007.pdf" >SimGIC</a>) combined with <a href="https://www.researchgate.net/profile/Jer-Hayes/publication/220837848_An_Intrinsic_Information_Content_Metric_for_Semantic_Similarity_in_WordNet/links/0fcfd50a2122fa6910000000/An-Intrinsic-Information-Content-Metric-for-Semantic-Similarity-in-WordNet.pdf" >ICSeco</a> approach;
			  (ii) one based on graph embedding methods (<a href="https://madoc.bib.uni-mannheim.de/41307/1/Ristoski_RDF2Vec.pdf" >RDF2Vec</a>).
			  </p>		
            </div>
          </div>

          <div class="col-lg-4 col-md-6 mt-4 mt-lg-0" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box iconbox-pink">
              <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,521.0016835830174C376.1290562159157,517.8887921683347,466.0731472004068,529.7835943286574,510.70327084640275,468.03025145048787C554.3714126377745,407.6079735673963,508.03601936045806,328.9844924480964,491.2728898941984,256.3432110539036C474.5976632858925,184.082847569629,479.9380746630129,96.60480741107993,416.23090153303,58.64404602377083C348.86323505073057,18.502131276798302,261.93793281208167,40.57373210992963,193.5410806939664,78.93577620505333C130.42746243093433,114.334589627462,98.30271207620316,179.96522072025542,76.75703585869454,249.04625023123273C51.97151888228291,328.5150500222984,13.704378332031375,421.85034740162234,66.52175969318436,486.19268352777647C119.04800174914682,550.1803526380478,217.28368757567262,524.383925680826,300,521.0016835830174"></path>
                </svg>
                <i class="bi bi-ui-checks"></i>
              </div>
			  <h4><a href="">Supervised Learning</a></h4>
              <p style="text-align: justify;"> Apply six well-known classes of machine learning models to train classifiers: 
			  <a href="https://ieeexplore.ieee.org/abstract/document/1053964" >K-Nearest Neighbor</a> (KNN);
			  <a href="https://link.springer.com/chapter/10.1007/0-387-28356-0_5" >Genetic Programming</a> (GP);
			  <a href="https://academic.oup.com/biomet/article-abstract/85/2/363/298822" >Decision Tree</a> (DT);
			  <a href="https://dl.acm.org/doi/abs/10.1145/2939672.2939785" >XGBoost</a> (XGB);
			  <a href="https://link.springer.com/article/10.1023/a:1010933404324" >Random Forest</a> (RF);
			  <a href="https://www.nature.com/articles/323533a0" >Multi-Layer Perceptron</a> (MLP).
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box iconbox-orange">
              <div class="icon">
                <svg width="100" height="100" viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg">
                  <path stroke="none" stroke-width="0" fill="#f5f5f5" d="M300,521.0016835830174C376.1290562159157,517.8887921683347,466.0731472004068,529.7835943286574,510.70327084640275,468.03025145048787C554.3714126377745,407.6079735673963,508.03601936045806,328.9844924480964,491.2728898941984,256.3432110539036C474.5976632858925,184.082847569629,479.9380746630129,96.60480741107993,416.23090153303,58.64404602377083C348.86323505073057,18.502131276798302,261.93793281208167,40.57373210992963,193.5410806939664,78.93577620505333C130.42746243093433,114.334589627462,98.30271207620316,179.96522072025542,76.75703585869454,249.04625023123273C51.97151888228291,328.5150500222984,13.704378332031375,421.85034740162234,66.52175969318436,486.19268352777647C119.04800174914682,550.1803526380478,217.28368757567262,524.383925680826,300,521.0016835830174"></path>
                </svg>
                <i class="bi bi-ui-checks"></i>
              </div>
			  <h4><a href="">Performance Evaluation</a></h4>
              <p style="text-align: justify;"> For evaluating the quality of a predicted classification, the <a href="https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html" >Weighted Average F-measure</a> (WAF) was used for stratified 10-fold cross-validation. The values ​​reported are the WAFs median.</p>
            </div>
          </div>

        </div>

      </div>
  </section><!-- End Methodology Section -->



<!-- ======= Results Section ======= -->
  <section id="results" class="results">
      <div class="container" data-aos="fade-up">
		
        <div class="section-title">
          <h2> Results </h2>
          <div class="row justify-content-center">
            <div class="col-8" >
              <span> We conducted two types of experiments: </span>
			  <span> <strong>(i) Same version</strong>, where we train the model with randomly chosen 10 000 protein interacting pairs from the archived STRING version and test it with the remaining pairs;</span>
			  <span> <strong>(ii) Future version</strong>, where we train the model with randomly chosen 10 000 protein pairs from the archived STRING version and test it on data from the current STRING version (excluding interactions present in the archived version).</span>
			  <span> The same randomly chosen 10 000 protein pairs are used in both settings.</span>
            </div>
          </div>
        </div>


		<div class="row justify-content-center">
          <div class="col-4">
            <div id="classifierFilter_div"></div>
          </div>
          <div class="col-4">
            <div id="ssmFilter_div"></div>
          </div>
        </div>


		<div class="row" >
          <div class="row justify-content-center">
            <div class="col-8" >
              <div id="table_div"></div>
            </div>
          </div>
		 
          <style>
           .cssSelectedTableRow {
             font-size: 15px;
             font-weight:bold;
           }
           .cssHeaderCell {
             font-size: 15px;
             border: solid 1px #FFFFFF;
           }
           .cssTableCell {
             font-size: 15px;
             border: solid 1px #FFFFFF;
           }
		   #table_div{
             width:  1310px;
             height: 380px;
             margin: 0; 
		   }
          </style>

          <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
          <script type="text/javascript">
          google.charts.load('current', {'packages':['corechart', 'table', 'controls']});
          google.charts.setOnLoadCallback(drawMainDashboard);

          function drawMainDashboard() {

            var dashboard = new google.visualization.Dashboard(
            document.getElementById('dashboard_div'));

            var classifierFilter = new google.visualization.ControlWrapper({
            'controlType': 'CategoryFilter',
            'containerId': 'classifierFilter_div',
            'options': {
            'filterColumnIndex': 0,
            'ui': {
            'caption': 'All Classifiers',
            'labelStacking': 'horizontal',
            'label': 'Choose a classifier:',
            'allowTyping': false,
            'allowMultiple': false
                  }
                }
             });

            var ssmFilter = new google.visualization.ControlWrapper({
            'controlType': 'CategoryFilter',
            'containerId': 'ssmFilter_div',
            'options': {
            'filterColumnIndex': 1,
            'ui': {
            'caption': 'All Similarity Measures',
            'labelStacking': 'horizontal',
            'label': 'Choose a similarity measure:',
            'allowTyping': false,
            'allowMultiple': false
                  }
                },
			'state': {'selectedValues': ['ResnikMax']}
             });

            var cssClassNames = {
            'selectedTableRow': 'cssSelectedTableRow',
            'headerCell': 'cssHeaderCell',
            'tableCell': 'cssTableCell',
            };

            var table2 = new google.visualization.ChartWrapper({
            'chartType': 'Table',
            'containerId': 'table_div',
            'options': {'width': '66%', 'height': 360, 'cssClassNames':cssClassNames},
            });

            var data2 = google.visualization.arrayToDataTable([
            ['Classifier', 'Measure', 'Same version 01/19', 'Future version 01/19', 'Same version 05/17', 'Future version 05/17', 'Same version 04/15', 'Future version 04/15'],
            ['KNN','ResnikMax',0.905,0.892,0.877,0.891,0.853,0.861],
			['KNN','SimGIC',0.858,0.843,0.821,0.842,0.825,0.826],
			['KNN','RDF2Vec',0.832,0.813,0.788,0.806,0.815,0.800],
			['GP','ResnikMax',0.896,0.893,0.877,0.888,0.856,0.843],
			['GP','SimGIC',0.873,0.855,0.835,0.859,0.835,0.842],
			['GP','RDF2Vec',0.848,0.829,0.813,0.830,0.836,0.823],
			['DT','ResnikMax',0.900,0.880,0.863,0.875,0.855,0.852],
			['DT','SimGIC',0.815,0.800,0.768,0.788,0.767,0.772],
			['DT','RDF2Vec',0.784,0.767,0.735,0.753,0.761,0.749],
			['XGB','ResnikMax',0.920,0.903,0.887,0.906,0.874,0.880],
			['XGB','SimGIC',0.873,0.858,0.839,0.860,0.834,0.846],
			['XGB','RDF2Vec',0.851,0.830,0.812,0.831,0.837,0.821],
			['RF','ResnikMax',0.912,0.902,0.885,0.902,0.867,0.880],
			['RF','SimGIC',0.874,0.858,0.837,0.859,0.832,0.845],
			['RF','RDF2Vec',0.851,0.830,0.813,0.831,0.838,0.822],
			['MLP','ResnikMax',0.902,0.894,0.880,0.896,0.860,0.869],
			['MLP','SimGIC',0.871,0.857,0.838,0.861,0.835,0.845],
			['MLP','RDF2Vec',0.851,0.829,0.813,0.831,0.839,0.823]
            ]);

            dashboard.bind([classifierFilter, ssmFilter], [table2, table2]);
            dashboard.draw(data2);
            }

          </script>
		</div>


        <div class="row" >
          <div class="row justify-content-center">
            <div class="col-8" >
		      <div id="boxplot_div"></div>
            </div>
		  </div>
		
		  <script src="https://cdn.anychart.com/releases/8.8.0/js/anychart-base.min.js"></script>
          <style type="text/css">
            #boxplot_div{
            width:  100%;
            height: 650px;
            margin: 0;
            padding-top: 20px;
		    }
          </style>
         
	      <script>
	  
          anychart.onDocumentReady(function () {
  
          var data1 = [
           {x:"All", low: 0.7351575811547773, q1: 0.8263387301741768, median: 0.8435610400845482, q3: 0.873324187080827, high: 0.9200569554545556},
	       {x:"ResnikMax", low: 0.8529089402412717, q1: 0.8641606785509511, median: 0.8785665892934128, q3: 0.8987648994667123, high: 0.9200569554545556},
	       {x:"SimGIC", low: 0.767201763369996, q1: 0.8264161960986258, median: 0.8349372308954044, q3: 0.8534712192662016, high: 0.8736131971487632},
	       {x:"RDF2Vec", low: 0.7351575811547773, q1: 0.812410210596547, median: 0.8235578406987907, q3: 0.838720824680285, high: 0.8511243823303396}
	       ];
	      var data2 = [
		   {x:"All", low: 0.7492308692542392, q1: 0.8268493070227108, median: 0.8451348587598732, q3: 0.8734398105265948, high: 0.9055119855369801},
		   {x:"ResnikMax", low: 0.8431668013948673, q1: 0.8761689132536509, median: 0.8894993745799793, q3: 0.895724860181129, high: 0.9055119855369801},
		   {x:"SimGIC", low: 0.7716923718444333, q1: 0.842251512018417, median: 0.8456474200394737, q3: 0.8582896394255981, high: 0.8605163598627782},
		   {x:"RDF2Vec", low: 0.7492308692542392, q1: 0.8081330858289864, median: 0.8228167094531215, q3: 0.8299351396240823, high: 0.8314043962297505}
	       ];

	      var chart = anychart.box();

	      series1 = chart.box(data1);
	      series1.name("Same version");

	      series1.normal().fill("#ff748c", 0.2);
	      series1.hovered().fill("#ff748c", 0.2);
	      series1.selected().fill("#ff748c", 0.6);
	      series1.normal().stroke("#ff748c", 1);
	      series1.hovered().stroke("#ff748c", 2);
	      series1.selected().stroke("#ff748c", 4);
	      series1.normal().medianStroke("#ff748c", 1);
	      series1.hovered().medianStroke("#ff748c", 2);
	      series1.selected().medianStroke("#ff748c", 2);
	      series1.normal().stemStroke("#ff748c", 0.5);
	      series1.hovered().stemStroke("#ff748c", 1);
	      series1.selected().stemStroke("#ff748c", 2);
	      series1.whiskerWidth(50);
	      series1.normal().whiskerStroke("#ff748c", 0.5);
	      series1.hovered().whiskerStroke("#ff748c", 1);
	      series1.selected().whiskerStroke("#ff748c", 2);

	      series2 = chart.box(data2);
	      series2.name("Future version");

	      series2.normal().fill("#0066cc", 0.2);
	      series2.hovered().fill("#0066cc", 0.2);
	      series2.selected().fill("#0066cc", 0.6);
	      series2.normal().stroke("#0066cc", 1);
	      series2.hovered().stroke("#0066cc", 2);
	      series2.selected().stroke("#0066cc", 4);
	      series2.normal().medianStroke("#0066cc", 1);
	      series2.hovered().medianStroke("#0066cc", 2);
	      series2.selected().medianStroke("#0066cc", 2);
	      series2.normal().stemStroke("#0066cc", 0.5);
	      series2.hovered().stemStroke("#0066cc", 1);
	      series2.selected().stemStroke("#0066cc", 2);
	      series2.whiskerWidth(50);
	      series2.normal().whiskerStroke("#0066cc", 0.5);
	      series2.hovered().whiskerStroke("#0066cc", 1);
	      series2.selected().whiskerStroke("#0066cc", 2);

	      chart.interactivity("by-x");
	  
	      var title = chart.title("Weighted Average F-measure Boxplot using the Same version and the Future version");
	      chart.title().fontSize(16);
	      chart.title().fontColor("#000000");	  
	      chart.title().fontWeight("bold");

	      chart.legend(true);
	      chart.legend().fontSize(15);
	      chart.legend().fontColor("#000000");	  
          chart.legend().fontWeight("bold");	  	  
	  
	      chart.yAxis().title("Weighted Average F-measure");
	      chart.yAxis().title().fontSize(15);
	      chart.yAxis().title().fontColor("#000000");	  
	 
	      chart.xAxis().labels().fontColor("#000000");
	      chart.xAxis().labels().fontSize(15);	

	      chart.yAxis().labels().fontColor("#000000");
	      chart.yAxis().labels().fontSize(15);
	  
	      chart.container("boxplot_div");
	      chart.draw();
	      });

	      </script>  

        </div>  

      </div> 
  </section><!-- End Results Section -->



 <!-- ======= Conclusions Section ======= -->
  <section id="conclusions" class="faq section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2> Conclusions </h2>
        </div>
        
		<div class="row justify-content-center">
		  <div class="col-8" >
            <div class="faq-list">
              <ul>
                <li data-aos="fade-up">
                  <i class="bx bx-check-circle icon-help"></i> <a> The results do not support a clear indication for data bias, indicating that if this problem exists, its magnitude is not affecting the performance of KG-based PPI predictions. </a> 
                </li>
                <li data-aos="fade-up" data-aos-delay="100">
                  <i class="bx bx-check-circle icon-help"></i> <a> The results indicate that the relation between the functions of a protein and its interactions do not fundamentally change over time. </a>
                </li>
              </ul>
            </div>
		  </div>
		</div>

      </div>
  </section><!-- End Conclusions Section -->



<!-- ======= About Video Section ======= -->
  <section id="minute-video" class="about-video">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>One-Minute Video</h2>
        </div>

        <div class="row justify-content-center">
          <img src="assets/img/OneMinuteVideo.png" class="img-fluid" alt="">
          <a href="https://www.youtube.com/watch?v=M22fEFDQSm0" class="glightbox play-btn mb-4" data-vbtype="video" data-autoplay="true"></a>
        </div>

      </div>
  </section><!-- End About Video Section -->



<!-- ======= Authors Section ======= -->
  <section id="authors" class="team section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Authors</h2>
        </div>

        <div class="row justify-content-center">

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="fade-up" data-aos-delay="100">
            <div class="member">
              <div class="member-img">
                <img src="assets/img/authors/Rita.png" class="img-fluid" alt="">
                <div class="social">
                  <a href="https://scholar.google.com/citations?user=Mfnwam0AAAAJ&hl"><i class="bi bi-google"></i></a>
                  <a href="mailto:risousa@ciencias.ulisboa.pt"><i class="bi bi-envelope"></i></a>
                  <a href="https://twitter.com/RitaTorresSousa"><i class="bi bi-twitter"></i></a>
                </div>
              </div>
              <div class="member-info">
                <h4> Rita T. Sousa </h4>
                <span>LASIGE, Faculdade de Ciências </span>
              </div>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="fade-up" data-aos-delay="200">
            <div class="member">
              <div class="member-img">
                <img src="assets/img/authors/ProfSara.jpg" class="img-fluid" alt="">
                <div class="social">
                  <a href="https://scholar.google.com/citations?user=g15oqi0AAAAJ&hl"><i class="bi bi-google"></i></a>
                </div>
              </div>
              <div class="member-info">
                <h4> Sara Silva </h4>
                <span>LASIGE, Faculdade de Ciências</span>
              </div>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch mb-5 mb-lg-0" data-aos="fade-up" data-aos-delay="300">
            <div class="member">
              <div class="member-img">
                <img src="assets/img/authors/ProfCatia.jpg" class="img-fluid" alt="">
                <div class="social">
                  <a href="https://scholar.google.com/citations?user=8nGYegMAAAAJ&hl"><i class="bi bi-google"></i></a>
                  <a href="https://twitter.com/CPesquita"><i class="bi bi-twitter"></i></a>
                </div>
              </div>
              <div class="member-info">
                <h4> Catia Pesquita </h4>
                <span>LASIGE, Faculdade de Ciências</span>
              </div>
            </div>
          </div>

        </div>

      </div>
  </section><!-- End Contact Section -->



<!-- ======= Funding Section ======= -->
  <section id="funding" class="funding">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2> Funding </h2>
          <p> Catia Pesquita, Sara Silva, Rita T. Sousa are funded by the FCT through LASIGE Research Unit, ref. UIDB/00408/2020 and ref. UIDP/00408/2020.
          Catia Pesquita and Rita T. Sousa are funded by project SMILAX (ref. PTDC/EEI-ESS/4633/2014), Sara Silva by projects BINDER (ref. PTDC/CCI-INF/29168/2017) 
          and PREDICT (ref. PTDC/CCI-CIF/29877/2017), and Rita T. Sousa by FCT PhD grant (ref. SFRH/BD/145377/2019). It was also partially supported by the KATY
          project which has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement No 101017453. </p> 
        </div>

        <div class="row justify-content-center">

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="100">
            <img src="assets/img/logos/ulisboa.png" class="img-fluid" alt="">
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="200">
            <img src="assets/img/logos/fcul.png" class="img-fluid" alt="">
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="300">
            <img src="assets/img/logos/fct.png" class="img-fluid" alt="">
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="400">
            <img src="assets/img/logos/lasige.jpg" class="img-fluid" alt="">
          </div>

        </div>

     </div>
  </section><!-- End Funding Section -->



  </main><!-- End #main -->



<!-- ======= Footer ======= -->
  <footer id="footer">

    <div class="footer-top">
      <div class="container">
        <div class="row"></div>
      </div>
    </div>

    <div class="container d-md-flex py-4">
      <div class="me-md-auto text-center text-md-start">
        <div class="credits">
          <!-- All the links in the footer should remain intact. -->
          <!-- You can delete the links only if you purchased the pro version. -->
          <!-- Licensing information: https://bootstrapmade.com/license/ -->
          <!-- Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/onepage-multipurpose-bootstrap-template/ -->
          Designed by <a href="https://bootstrapmade.com/">BootstrapMade</a>
        </div>
      </div>
    </div>

  </footer>
<!-- End Footer -->

    <div id="preloader"></div>
    <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>

<!-- Vendor JS Files -->
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>
  <script src="assets/vendor/purecounter/purecounter.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>

<!-- Template Main JS File -->
<script src="assets/js/main.js"></script>

</body>

</html>
